src/App.js

```diff
@@ -33,6 +35,7 @@ function App(props) {
   ReactGA.pageview(window.location.pathname + window.location.search);
   return (
     <ThemeProvider theme={theme}>
+      <CssBaseline />
```

Comment:
Thanks for adding this -- it's a standard practice to include this on sites/web apps nowadays but we've neglected this (until now)